donetick/docker-compose.yml

```
@@ -0,0 +1,17 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: donetick_app_1
+      APP_PORT: 2021

```

We can disable the proxy auth here as Donetick comes with authentication out of the box.
```suggestion
      APP_PORT: 2021
      PROXY_AUTH_ADD: "false"

```